### PR TITLE
Rc 0.8.0 docs links

### PR DESCRIPTION
**Pull request by bcolloran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ or try our example:
 ```
 rill init-example
 ```
-(Please note that the command `rill init-example` is temporarily unavailable on Windows.)
+<!-- (Please note that the command `rill init-example` is temporarily unavailable on Windows.) -->
```
Comment (magorlick) [inline README.md:48]: should we keep this for people on working versions using windows (I talked to one yesterday, I know they exist!)

File: docs/docs/README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ You can [file an issue](https://github.com/rilldata/rill-developer/issues/new/ch
 ## Pick an install option:
 Our binary is the fastest way to install Rill Developer:
 - [binary](https://docs.rilldata.com/install/binary) : download the most recent assets binary.
-  - [macos-arm64](https://storage.googleapis.com/pkg.rilldata.com/rill-developer-example/binaries/0.7/macos-arm64/rill)
-  - [macos-x64](https://storage.googleapis.com/pkg.rilldata.com/rill-developer-example/binaries/0.7/macos-x64/rill)
-  - [linux-x64](https://storage.googleapis.com/pkg.rilldata.com/rill-developer-example/binaries/0.7/linux-x64/rill)
-  - [win-x64](https://storage.googleapis.com/pkg.rilldata.com/rill-developer-example/binaries/0.7/win-x64/rill.exe)
+  - [macos-arm64](https://storage.googleapis.com/pkg.rilldata.com/rill-developer-example/binaries/0.8/macos-arm64/rill)
```
Comment (magorlick) [inline docs/docs/README.md:27]: None of these new links work for me - did you upload them to the CDN?

File: docs/docs/README.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ rill init-example
 
 ```
 
-(Please note that the command `rill init-example` is temporarily unavailable on Windows.)
+<!-- (Please note that the command `rill init-example` is temporarily unavailable on Windows.) -->
```
Comment (magorlick) [inline docs/docs/README.md:55]: should we keep this for people on working versions using windows

File: docs/docs/cli.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ or try our example:
 ```
 rill init-example
 ```
-(Please note that the command `rill init-example` is temporarily unavailable on Windows.)
+<!-- (Please note that the command `rill init-example` is temporarily unavailable on Windows.) -->
```
Comment (magorlick) [inline docs/docs/cli.md:23]: should we keep this for people on working versions using windows

**Review by magorlick — CHANGES_REQUESTED** [review]
I don't see the release notes in this PR.  Are you doing those in a different PR?

**Review by hamilton — APPROVED** [review]

**Pull request merged**